tests/baselines/reference/superPropertyAccess2.errors.txt

```diff
@@ -28,8 +27,6 @@ superPropertyAccess2.ts(24,15): error TS2339: Property 'x' does not exist on typ
                   ~~~
 !!! error TS2576: Property 'bar' does not exist on type 'C'. Did you mean to access the static member 'C.bar' instead?
             super.x;  // error
-                  ~
-!!! error TS2340: Only public and protected methods of the base class are accessible via the 'super' keyword.
         }

```

Comment:
Suspicious missing error; should still error in newer ecmascript

Reply:
Due to `Only_public_and_protected_methods_of_the_base_class_are_accessible_via_the_super_keyword` only firing in `<ES2015`.